addon/mixins/popup.js

```diff
@@ -63,7 +63,7 @@ export default Mixin.create({
   didCreateLayer() {
     this._super(...arguments);
     if (this.get('hasBlock')) {
-      this._popup = this.L.popup({}, this._layer);
+      this._popup = this.L.popup(this.get('popupOptions'), this._layer);
       this._popup.setContent(this.get('destinationElement'));
       this._layer.bindPopup(this._popup, this.get('popupOptions'));
```

Comment:
Let's try having the options *only on popup creation* and see what comes out. /cc @multiplegeorges